pkgs/top-level/all-packages.nix

```diff
@@ -8556,6 +8556,10 @@ with pkgs;
 
   gparted = callPackage ../tools/misc/gparted { };
 
+  gpt4all-cuda = gpt4all.override {
```

Comment:
I dont think we should add this. Just override `gpt4all` and it's enough.

Reply:
Hmm, we have a bunch of similarly constructed `*-cuda` packages. I thought it was a correct approach 🤔 .

Reply:
Oh that is something new for me then.